frontend/src/common/models.ts

```diff
@@ -279,20 +279,25 @@ export interface IApartmentWritable {
 
 // //  Maximum Price Fields
 
-interface ICalculation {
+export interface ICalculation {
     maximum_price: number;
     valid_until: string;
     maximum: boolean;
 }
 
-interface IIndexCalculationVariables {
+export interface IIndexCalculationVariables {
     calculation_variables: {
         acquisition_price: number;
         additional_work_during_construction: number;
         basic_price: number;
         index_adjustment: number;
         apartment_improvements: number;
-        housing_company_improvements: number;
+        housing_company_improvements: {
+            summary: {
+                value: number;
+                value_for_apartment: number;
+            };
+        };
```

Comment:
Missing some fields, but we can add them when they become necessary, the json is not fully done in the backend yet anyway :)

Reply:
Yep, my thoughts exactly.